demo/app/assets/javascripts/primer.js

```diff
@@ -0,0 +1 @@
+import '../../../../app/assets/javascripts/primer.js';
```

Comment:
Does this work? I think this needs to be a symlink, or we need the demo package to depend on the root package as a gem

Reply:
so, the demo does depend on `primer_view_components` gem, but yeah it does not look like it is working as-is 🤔